examples/with-ramda/index.js

```diff
@@ -45,17 +44,17 @@ const buildResponse = R.cond([
   [equals('Whatchyu doing?'), always('nothing much, how about you?')],
   [
     T,
-    temp => 'So what are you really saying?'
-  ] /* this last array contains the default response for your bot */
-])
+    () => 'So what are you really saying?',
+  ] /* this last array contains the default response for your bot */,
```

Comment:
should we move the comma `,` right after closing bracket `]` like `], /* xxx */`?